src/dan/Dan.jsx

```diff
@@ -1,38 +1,168 @@
+import React from 'react'
+import myImage from '../assets/e948592c6c5a1022f18fe18d527af199.png'
 
-import "./Dan.css";
-import Navbar from "./components/Navbar";
-import About from "./components/About";
-import Projects from "./components/Projects";
-import Home from "./components/Home";
-import 'bootstrap/dist/css/bootstrap.min.css';
-import Contact from "./components/Contact";
-import Experience from "./components/Experience";
+import './Dan.css'
 
-function Dan() {
-  return (
-    <div className="App">
-      <Navbar />
-      <div id="home">
-        <Home />
-      </div>
-
-      <div id="about">
-        <About />
-      </div>
+const Dan = () => {
```

Comment:
Make your component Reusable, so it takes props. And rename it to actual component name.